config/environments/test.rb

```diff
@@ -10,6 +10,8 @@
   config.action_controller.perform_caching = false
 
   config.action_dispatch.show_exceptions = false
+  config.action_mailer.default_url_options = {:host => "localhost:3000"}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.